apps/expo/src/components/ui/dialog.tsx

```
@@ -40,18 +42,24 @@ const DialogOverlayNative = React.forwardRef<
+        "z-50 flex bg-black/80 justify-center items-center p-2",
+        className,
+      )}
       {...props}
       ref={ref}
     >
-      <Animated.View entering={FadeIn.duration(150)} exiting={FadeOut.duration(150)}>
+      <Animated.View
+        entering={FadeIn.duration(150)}
+        exiting={FadeOut.duration(150)}
+      >
         <>{children}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)